src/heliaServer.ts

```diff
@@ -94,6 +100,10 @@ export class HeliaServer {
         path: '/',
         type: 'GET',
         handler: async (request, reply): Promise<void> => {
+          if (request.url.includes('/api/v0')) {
+            await reply.code(400).send('API + Method not supported')
+            return
+          }
```

Comment:
Trying to catch and prevent the above mentioned issue where calls that shouldn't be caught here are.